grails-app/conf/Config.groovy

```diff
@@ -917,7 +917,7 @@ openboxes {
         }
         analytics {
             enabled = true
-            requiredRoles = [RoleType.ROLE_ADMIN]
+            requiredRoles = [RoleType.ROLE_SUPERUSER, RoleType.ROLE_ADMIN]
```

Comment:
@awalkowiak How does the requiredRoles work? Is it minimum required role? Or any of these roles? Based on your response, I'll let you decide whether this one should be merged or changed?

Reply:
@jmiranda there is used hasAnyRoles, so this usage is ok. But I am wondering if the minimum required role would not be better (cleaner)

Reply:
As it goes to hotfix then I am merging as it is